quix-frontend/client/src/lib/sql-autocomplete/adapter/sql-autocpmplete-adapter-utills.ts

```diff
@@ -0,0 +1,304 @@
+import { TableInfo } from "../sql-context-evaluator";
+
+interface ObjectChild {
+  name: string;
+  dataType: any;
+}
+
+export function getObjectChildren(obj: Record<string, any>, parentName = ''): ObjectChild[] {
+  const children: ObjectChild[] = [];
+
+  for (const key in obj) {
+    if (typeof obj[key] === 'object' && obj[key] !== null) {
+      const childName = parentName ? `${parentName}.${key}` : key;
+      children.push({ name: childName, dataType: obj[key] });
```

Comment:
it is duplication of 17-18.
you should do it once

Reply:
fixed!